src/components/nav-button.svelte

```diff
@@ -65,7 +66,8 @@ border-white
   xl:items-center md:items-start border-l-4 lg:border-l-0 
   border-{$activePath === link ? text : background} lg:border-{background} 
   lg:shadow-{$activePath === link ? background : 'none'} focus:underline"
-  href={link}>
+  href={link}
+  on:click={click}>
```

Comment:
Tässä on mahdollista jättää tuo click-handleri propdrillaamatta näin syvälle ja antaa tälle vain on:click direktiivi. Siten tuon handlauksen voi hoitaa jossain parenttikomponentissa samalla direktiivillä.

Reply:
on:click komponentti elementissä (NavButton) ei näytä toimivan, pitää olla itse a-elementissä

Reply:
outoa, kyllä sen tietääkseni pitäisi bubblea jos laittaa vaan pelkän on:click direktiivin.